py_pdf_parser/tables.py

```diff
@@ -431,3 +443,45 @@ def _fix_cols(cols: Set["ElementList"], elements: "ElementList") -> None:
             else:
                 sorted_columns.remove(col)
     return
+
+
+def _remove_duplicate_header_rows(table: List[List[Any]]) -> List[List[Any]]:
+    """
+    Removes rows which are duplicates of the header (i.e., the first) row.
+    A row is considered duplicate if all of its elements have the same text and font of
+    their correspondent elements (i.e., same index) in the header row.
+
+    Args:
+        table (List[List[Any]]): The table to remove the duplicate headers from.
+
+    Returns:
+        List[List[Any]]: The table without the duplicate header rows.
+
+    """
+    if len(table) <= 1:
+        return table
+
+    def elements_equal(elem_1: Optional["PDFElement"], elem_2: Optional["PDFElement"]):
+        if elem_1 is None and elem_2 is None:
+            return True
+
+        if (elem_1 is None or elem_2 is None) or (
+            elem_2 is None and elem_1 is not None
+        ):
+            return False
```

Comment:
Given the above check, we know they're not BOTH `None`, so I think this can be simplified to `if elem1 is None or elem2 is None`?

Reply:
Yeah I did that originally, but I thought it would be better to be explicit for readability purposes. But I guess you're right, it will be readable enough, I am going to change it :+1: